lib/crawler/data/crawl_result/html.rb

```diff
@@ -26,6 +26,19 @@ def parsed_content
           @parsed_content ||= Jsoup.parse(content)
         end
 
+        def parsed_content_excluding_tags(exclude_tags)
```

Comment:
As I understand it, `parsed_content` atm is used both for crawling, to extract more links for next iterations, and for indexing. I created a separate, stripped out content attribute, so that we make sure the 2 aren't getting mixed up.